### PR TITLE
KOGITO-8291 Setup Prod Jenkins check

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-8291

### CODE REVIEW

**Comment by kie-ci1** [comment]
**PR job** `#360` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/display/redirect

**Test results:**
- PASSED: 243
- FAILED: 6

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/testReport/features.image.common-dynamic-resources/Common tests for Kogito images/Build___Test_Images___Build_Test_kogito_jobs_service_infinispan___Build/Test_kogito_jobs_service_infinispan___Verify_if_the_properties_were_correctly_set_using_DEFAULT_MEM_RATIO/">Build & Test Images / Build&Test kogito-jobs-service-infinispan / Build/Test kogito-jobs-service-infinispan / features.image.common-dynamic-resources.Common tests for Kogito images.Verify if the properties were correctly set using DEFAULT MEM RATIO</a></summary>
HOOK-ERROR in after_scenario: APIError: 409 Client Error for http+docker://localhost/v1.22/containers/eadde7cc8d8eddc11cb8b1e187ce0ab8c91cafad5e644c6cb879e6ae2b9d7012/kill: Conflict ("b'Cannot kill container: eadde7cc8d8eddc11cb8b1e187ce0ab8c91cafad5e644c6cb879e6ae2b9d7012: Container eadde7cc8d8eddc11cb8b1e187ce0ab8c91cafad5e644c6cb879e6ae2b9d7012 is not running'")
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_using_native_build_and_runtime_image/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected using native build and runtime image</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_using_native_build_and_no_runtime_image/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected using native build and no runtime image</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_performing_a_native_build_and_if_it_is_listening_on_the_expected_port,_test_uses_custom_properties_file_to_test_the_port_configuration./">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected performing a native build and if it is listening on the expected port, test uses custom properties file to test the port configuration.</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_performing_a_native_build_with_persistence_enabled___Step_1:_build_the_application_and_copy_to_the_runtime_image/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected performing a native build with persistence enabled - Step 1: build the application and copy to the runtime image</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/360/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Perform_a_second_incremental_s2i_build_for_native_scenario,_this_time,_with_native_enabled/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Perform a second incremental s2i build for native scenario, this time, with native enabled</a></summary>
S2I build failed, check logs!
</details>

**Comment by kie-ci1** [comment]
**PR job** `#361` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/361/display/redirect

**Test results:**
- PASSED: 244
- FAILED: 5

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/361/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_using_native_build_and_runtime_image/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected using native build and runtime image</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/361/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_using_native_build_and_no_runtime_image/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected using native build and no runtime image</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/361/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_performing_a_native_build_and_if_it_is_listening_on_the_expected_port,_test_uses_custom_properties_file_to_test_the_port_configuration./">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected performing a native build and if it is listening on the expected port, test uses custom properties file to test the port configuration.</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/361/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Verify_if_the_s2i_build_is_finished_as_expected_performing_a_native_build_with_persistence_enabled___Step_1:_build_the_application_and_copy_to_the_runtime_image/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Verify if the s2i build is finished as expected performing a native build with persistence enabled - Step 1: build the application and copy to the runtime image</a></summary>
S2I build failed, check logs!
</details>
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/361/testReport/features.image.kogito-s2i-builder-native/kogito-s2i-builder image native build tests/Perform_a_second_incremental_s2i_build_for_native_scenario,_this_time,_with_native_enabled/">features.image.kogito-s2i-builder-native.kogito-s2i-builder image native build tests.Perform a second incremental s2i build for native scenario, this time, with native enabled</a></summary>
S2I build failed, check logs!
</details>

**Comment by spolti** [comment]
@radtriste can we have a jira for this?

**Review by spolti — COMMENTED** [review]
don't we need to update the image override files as well?

**Comment by radtriste** [comment]
@spolti I just want to reflect what we are doing in prod, which means we don't build the `rhpam-` images but the `logic-` ones
For now, there is no update of `rhpam-` images. They are kept as history I guess.

**Comment by spolti** [comment]
It's fine by me, but we need to decide how we are going to handle the rhpam image for this repo, if we are not going to build and use it anymore, it is probably better to remove from main.

**Comment by radtriste** [comment]
I would create a follow-up issue to deal with those `rhpam-` images. 
We could set them as deprecated for now.

**Comment by spolti** [comment]
> I would create a follow-up issue to deal with those `rhpam-` images. We could set them as deprecated for now.

 That's a good plan :)
Thanks.

**Review by spolti — APPROVED** [review]

**Comment by radtriste** [comment]
Jenkins run Prod

**Comment by radtriste** [comment]
Jenkins run Prod

**Comment by radtriste** [comment]
Jenkins run Prod

File: .ci/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -31,9 +31,14 @@ pipeline {
                     if (env.MAVEN_MIRROR_REPOSITORY != null
                             && env.MAVEN_MIRROR_REPOSITORY != '') {
                         env.MAVEN_MIRROR_URL = env.MAVEN_MIRROR_REPOSITORY
-                    }
+                            }
```
Comment (spolti) [inline .ci/jenkins/Jenkinsfile]: is the indentation correct?

Comment (radtriste) [reply .ci/jenkins/Jenkinsfile]: it is not ...

**Comment by radtriste** [comment]
jenkins run prod

**Review by spolti — APPROVED** [review]

**Review by desmax74 — APPROVED** [review]

**Comment by radtriste** [comment]
jenkins retest this

**Comment by radtriste** [comment]
jenkins run prod

**Pull request merged**